src/internal/internetReachability.ts

```diff
@@ -79,26 +79,19 @@ export default class InternetReachability {
 
     // Create promise that will reject after the request timeout has been reached
     let timeoutHandle: ReturnType<typeof setTimeout>;
-    const timeoutPromise = new Promise<Response>(
-      (_, reject): void => {
-        timeoutHandle = setTimeout((): void => {
-          controller.abort();
-          reject('timedout');
-        }, this._configuration.reachabilityRequestTimeout);
-      },
-    );
+    const timeoutPromise = new Promise<Response>((): void => {
+      timeoutHandle = setTimeout(
+        (): void => controller.abort('timedout'),
+        this._configuration.reachabilityRequestTimeout,
+      );
+    });
```

Comment:
You have merged main into this PR and then reformatted it.
So to me it looks like you have changed these files, but you have not.
As a reviewer, that is troubling.

Can you revert any changes to files that are not necessary for your specific feature?

Reply:
I could not revert changes. So I've created new branch and pull request [#703](https://github.com/react-native-netinfo/react-native-netinfo/pull/703)

Reply:
thanks - I recognize that was kind of a pain for such a small change but as a maintainer + reviewer doing this for zero compensation just as a volunteer in my spare time, it really *really* helps to have clean diffs